terra-env/variables.tf

```diff
@@ -41,11 +41,26 @@ variable "owner" {
 locals {
   owner = var.owner == "" ? terraform.workspace : var.owner
 }
+
+
+#
+# Preview environment Vars
```

Comment:
It might be nice to have some documentation here or linked to explaining what a preview environment is and how to tell if your resource should exist always, for preview environments, or for preview_shared environments.